docs/quickstart.rst

```diff
@@ -99,7 +101,7 @@ we can evaluate metrics to get a group summary as below:
     DecisionTreeClassifier(...)
     >>> y_pred = classifier.predict(X)
     >>> group_summary(accuracy_score, y_true, y_pred, sensitive_features=sex)
-    {'overall': 0.844..., 'by_group': {'female': 0.925..., 'male': 0.804...}}
+    {'overall': 0.8443552680070431, 'by_group': {'Female': 0.9251482213438735, 'Male': 0.8042879019908117}}
```

Comment:
Doesn't the ellipsis force the match to be approximate when these are run as tests? Why give pointless precision?

Reply:
Per:
https://github.com/fairlearn/fairlearn/pull/488
the ellipsis is a _text_ wildcard. So you can truncate the decimals, but don't round when doing so.

Reply:
Thanks for the pointer! Updated accordingly.